### PR TITLE
#38 Custom tooltip color and tooltip text color.

### LINKED ISSUE #38: Custom the tooltips color through the config object
## context
It is not possible to custom the tooltips color, the idea is to set a color to the tooltip and perhaps to the text as well.

### PR DESCRIPTION
**Pull request by MarioRodriguezS**
## Description
To customize the tooltips was necessary to implement a Mutation Observer because the tooltip tag rendered by editorjs is dynamic, and it changes the content when the tooltips are hovered with the mouse.  Also, in the case of the color tooltip, it was necessary to insert rules in the :before and :after pseudo-elements with JavaScript.

Fix #38

### CODE REVIEW

**Comment by juanhurtado10** [comment]
@MarioRodriguezS as this is a tool config, the Readme file should include these newly config options

**Comment by MarioRodriguezS** [comment]
> @MarioRodriguezS as this is a tool config, the Readme file should include these newly config options

Sure! I forgot it.

File: README.md [hunk 1]
```diff
@@ -53,6 +56,8 @@ Select some text, click on the tooltip button in the inline-tools bar, and type
 | location          | `{location: string}` | You could choose the tooltip place(top, bottom, left, right). Default value is bottom
 | color | `{ color: Hex or string }`|You could choose the background text color wrapped by the tooltip, you could pass hexadecimal colors or string colors
 | underline | `{ underline: boolean }`  | You could add underline text decoration to the text wrapped by the tooltip
+| tooltipColor | `{ tooltipColor: string }`  | You could choose the tooltip color, you could pass hexadecimal colors or string colors
```
Comment (juanhurtado10) [inline README.md:59]: @MarioRodriguezS not really sure but IMO, we should use more specific names like `backgroundColor`, `textColor` and `highlightColor`  What do you think?

**Review by juanhurtado10 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (MarioRodriguezS) [reply README.md]: @juanhurtado10  comments solved. Thanks!

**Review by juanhurtado10 — APPROVED** [review]

**Pull request merged**